src/status_im2/contexts/communities/requests/actions/view.cljs

```diff
@@ -10,7 +10,7 @@
             [status-im.communities.core :as communities]
             [status-im.i18n.i18n :as i18n]
             [status-im.react-native.resources :as resources]
-            [status-im.ui.components.list.views :as list]
+            [status-im.utils.datetime :as datetime]
```

Comment:
going to make a follow up p.r and address the rest of the imports.. didn't want to increase the size of this p.r